File-Formats/PDF/Working-with-OCR/aspnet-mvc.md

```diff
@@ -0,0 +1,101 @@
+---
+title: Perform OCR on a PDF document in ASP.NET MVC | Syncfusion
+description: Learn how to efficiently perform OCR on a PDF document in ASP.NET MVC with easy steps using Syncfusion .NET library.
+platform: file-formats
+control: PDF
+documentation: UG
+keywords: Assemblies
+---
+
+# Perform OCR on a PDF document in ASP.NET MVC
+
+The Syncfusion Essential PDF is a .NET PDF library that supports OCR by using the Tesseract open-source engine in ASP.NET MVC application.  
+
+## Steps to perform OCR on a PDF document in ASP.NET MVC
```

Comment:
Steps to perform OCR on entire PDF document in ASP.NET MVC

Reply:
changed the title